apis/r/NEWS.md

```diff
@@ -5,9 +5,7 @@
 * Add support for writing `SummarizedExperiment` and `SingleCellExperiment` object to SOMAs
 
 
-# tiledbsoma 1.4.0
-
-This is the initial release of the tiledbsoma R package.
```

Comment:
Hm, why remove an entry ?  That was a release, was it not?

Reply:
@johnkerl I read's Aaron's suggestion as proposing to also update NEWS.md.  Currently the PR only touches DESCRIPTION.

Reply:
I was wrong. This release will only include a compatibility fix in Python, nothing new from R. So we just need to update version in `DESCRIPTION` and backport that to 1.4.

Reply:
@eddelbuettel he changed his mind in Slack